mgmt/RecordsConfig.cc

```diff
@@ -1374,6 +1374,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.quic.client.keylog_file", RECD_STRING, nullptr , RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.quic.qlog_file", RECD_STRING, nullptr , RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
Why is this not a directory name nor a filename prefix? It seems like the filename is always the same and 2nd connection overwrites the existing file. What if two concurrent connections tried to dump logs at the same time? This is the reason I suggested to include timestamp or CID to filenames.